packages/client/tests/functional/extensions/itx.ts

```diff
@@ -315,7 +315,7 @@ testMatrix.setupTestSuite(
             if (isTransaction) {
               expect(ctx.$connect).toBeUndefined()
               expect(ctx.$disconnect).toBeUndefined()
-              expect(ctx.$transaction).toBeUndefined()
+              expect(ctx.$transaction).toBeDefined()
```

Comment:
This should be undefined on MongoDB

Reply:
@aqrln My preference would be to leave it defined so there is a consistent interface, and throw a readable error, like what is done for the planetscale adapter.